server/lib/db/index.js

```diff
@@ -115,8 +114,8 @@ class DatabaseConnector {
   one(sql, params, id, entity = 'record') {
     return this.exec(sql, params)
       .then(rows => {
-        const errorMessage =
-          `Expected ${entity} to contain a single record with id ${id}, but ${rows.length} were found!`;
+        // eslint-disable-next-line max-len
```

Comment:
😄 is it allowed ?

Reply:
Eh, it's either that or figure our a shorter representation ;)